src/ConvergenceChecker.cpp

```diff
@@ -82,8 +83,11 @@ void ConvergenceChecker::DefineAbsoluteTolerances() {
  * name: Name of unknown quantity to get default absolute tolerance for.
  */
 real_t ConvergenceChecker::GetDefaultAbsTol(const string &name) {
+    real_t abstol_nre = 1e-10;
     if (name == OptionConstants::UQTY_N_RE)
-        return 1e12;
+        return abstol_nre;
+    else if ( name == OptionConstants::UQTY_J_RE)
+        return Constants::ec*Constants::c*abstol_nre; 
```

Comment:
I made the j_re tolerance consistent with the n_re tolerance (which I also lowered a bit since I felt the default was a bit too coarse)